runners/samza/build.gradle

```diff
@@ -57,12 +58,12 @@ dependencies {
   runtimeOnly "org.rocksdb:rocksdbjni:6.15.2"
   runtimeOnly "org.scala-lang:scala-library:2.11.8"
   implementation "org.apache.samza:samza-api:$samza_version"
-  implementation "org.apache.samza:samza-core_2.11:$samza_version"
-  runtimeOnly "org.apache.samza:samza-kafka_2.11:$samza_version"
-  runtimeOnly "org.apache.samza:samza-kv_2.11:$samza_version"
-  implementation "org.apache.samza:samza-kv-rocksdb_2.11:$samza_version"
-  implementation "org.apache.samza:samza-kv-inmemory_2.11:$samza_version"
-  implementation "org.apache.samza:samza-yarn_2.11:$samza_version"
+  implementation "org.apache.samza:samza-core_$samza_scala:$samza_version"
```

Comment:
this may affect users, however given that samza 1.6.0 is 3 years old I would consider this more like a long overdue update.

tested validatesRunner still passes on both Java8 and 11

Reply:
Revert Samza changes because actually the version bump is non-trivial. Python Validates Runner Samza test now failing with various reasons. First it is

```
 java.lang.UnsupportedOperationException: SamzaImpulseSystem doesn't support producing
 	at org.apache.beam.runners.samza.translation.SamzaImpulseSystemFactory.getProducer(SamzaImpulseSystemFactory.java:59)
 	at org.apache.samza.system.SystemFactory.getProducer(SystemFactory.java:66)
 	at org.apache.samza.container.SamzaContainer$.$anonfun$apply$27(SamzaContainer.scala:248)
 	at scala.collection.TraversableLike.$anonfun$map$1(TraversableLike.scala:286)
 	...
 	at scala.collection.AbstractTraversable.map(Traversable.scala:108)
 	at org.apache.samza.container.SamzaContainer$.apply(SamzaContainer.scala:245)
 	at org.apache.samza.container.SamzaContainer.apply(SamzaContainer.scala)
 	at org.apache.samza.processor.StreamProcessor.createSamzaContainer(StreamProcessor.java:412)
```

after change `SamzaImpulseSystemFactory.getProducer` to return null (as other getProducer), tests then fail with

```
024-08-15T21:18:38.5903463Z [grpc-default-executor-1] INFO org.apache.beam.runners.fnexecution.control.FnApiControlClientPoolService - getProcessBundleDescriptor request with id 18-4
 Traceback (most recent call last):
   File "/runner/_work/beam/beam/sdks/python/test-suites/portable/py38/build/srcs/sdks/python/apache_beam/runners/common.py", line 1495, in process
     return self.do_fn_invoker.invoke_process(windowed_value)
   File "/runner/_work/beam/beam/sdks/python/test-suites/portable/py38/build/srcs/sdks/python/apache_beam/runners/common.py", line 687, in invoke_process
     self.output_handler.handle_process_outputs(
   File "/runner/_work/beam/beam/sdks/python/test-suites/portable/py38/build/srcs/sdks/python/apache_beam/runners/common.py", line 1681, in handle_process_outputs
     self._write_value_to_tag(tag, windowed_value, watermark_estimator)
   ...
   File "/runner/_work/beam/beam/sdks/python/test-suites/portable/py38/build/srcs/sdks/python/apache_beam/coders/coder_impl.py", line 1259, in get_estimated_size_and_observables
     self._elem_coder.get_estimated_size_and_observables(
   File "/runner/_work/beam/beam/sdks/python/test-suites/portable/py38/build/srcs/sdks/python/apache_beam/coders/coder_impl.py", line 291, in get_estimated_size_and_observables
     return self.estimate_size(value, nested), []
   File "/runner/_work/beam/beam/sdks/python/test-suites/portable/py38/build/srcs/sdks/python/apache_beam/coders/coder_impl.py", line 282, in estimate_size
     return self._get_nested_size(self._size_estimator(value), nested)
   File "/runner/_work/beam/beam/sdks/python/test-suites/portable/py38/build/srcs/sdks/python/apache_beam/coders/coders.py", line 209, in estimate_size
     return len(self.encode(value))
   File "/runner/_work/beam/beam/sdks/python/test-suites/portable/py38/build/srcs/sdks/python/apache_beam/coders/coders.py", line 407, in encode
     return value.encode('utf-8')
 AttributeError: 'numpy.int64' object has no attribute 'encode'
```